### PR TITLE
[#1305] Data script to add users, update old roles

### PR DESCRIPTION
**Pull request by muloem**
* Add users or update old roles in the datastore

### CODE REVIEW

**Comment by iperdomo** [comment]
## Checklist

* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting

File: scripts/data/src/org/akvo/gae/remoteapi/AddUsers.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        ds.put(users.values());
+    }
+
+    private Map<String, Entity> createUserEntites(List<String> userLines) {
+        Map<String, Entity> users = new HashMap<String, Entity>();
+
+        for (String line : userLines) {
+            String[] userParts = line.split(",", 3);
+            if (userParts.length < 3 || userParts[0].trim().length() == 0) {
+                System.out.println("Skipping user: " + line);
```
Comment (ichinaski) [inline scripts/data/src/org/akvo/gae/remoteapi/AddUsers.java]: I believe this is missing a `continue`?

Comment (muloem) [inline scripts/data/src/org/akvo/gae/remoteapi/AddUsers.java]: Good catch!

**Comment by ichinaski** [comment]
Code reviewed

**Pull request merged**